tests/helpers/common/NuGet/Get-NuGetPaths.ps1

```diff
@@ -0,0 +1,8 @@
+function Get-NugetPaths {

```

Comment:
@corbob I know it doesn't make any difference, but I am going to get this changed from `Get-NugetPaths` to `Get-NuGetPaths`.

Reply:
~~It should also be `Get-NuGetPath` because we don't do plurals in PowerShell but anyway~~